common/services/reference-data.js

```diff
@@ -92,6 +92,10 @@ const referenceDataService = {
     })
   },
 
+  getRegions() {
+    return apiClient.findAll('region').then(response => response.data)
```

Comment:
This might hit a snag if there are more than 100 regions as I believe that's the current API page size limit. We might need to do something more like the locations method to ensure we get all results.